### ISSUE TITLE
Could not find concurrent-ruby

### ISSUE THREAD

**Issue by elyashiv** [unused/exercism-docker#2]
When I run `make` I get the following error:

```
rails_1  | /usr/local/lib/ruby/site_ruby/2.5.0/bundler/spec_set.rb:91:in `block in materialize': Could not find concurrent-ruby-1.1.3 in any of the sources (Bundler::GemNotFound)
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/bundler/spec_set.rb:85:in `map!'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/bundler/spec_set.rb:85:in `materialize'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/bundler/definition.rb:171:in `specs'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/bundler/definition.rb:238:in `specs_for'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/bundler/definition.rb:227:in `requested_specs'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/bundler/runtime.rb:108:in `block in definition_method'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/bundler/runtime.rb:20:in `setup'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/bundler.rb:107:in `setup'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/bundler/setup.rb:20:in `<top (required)>'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
rails_1  | 	from /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
rails_1  | 	from /opt/rails/config/boot.rb:3:in `<top (required)>'
rails_1  | 	from bin/rails:3:in `require_relative'
rails_1  | 	from bin/rails:3:in `<main>'
exercism_rails_1 exited with code 1
```
Docker version is 18, I'm not sure what other data might be needed...

**Comment by unused** [comment]
TL;DR: should be fixed, thanks for the report!

Sorry for the late response :palm_tree: - there was some issue with the used ruby version and the gem concurrent ruby ... exercism did not define the ruby version before, that's why latest was used in the rails docker container.
